src/topics/tools.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
 
-var async = require('async'),
+var async = require('async');
 
-	db = require('../database'),
-	categories = require('../categories'),
-	plugins = require('../plugins'),
-	privileges = require('../privileges');
+var db = require('../database');
+var categories = require('../categories');
+var meta = require('../meta');
+var plugins = require('../plugins');
+var privileges = require('../privileges');
```

Comment:
coz the NodeBB style guide says so...
https://docs.nodebb.org/en/latest/contributing/style-guide.html